src/chatClient.cpp

```diff
@@ -3226,11 +3256,16 @@ promise::Promise<void> GroupChatRoom::autojoinPublicChat(uint64_t ph)
         std::string uKeyB64;
         mega::Base64::btoa(uKeyBin, uKeyB64);
 
+        parent.mKarereClient.setCommitMode(false);
```

Comment:
Not sure, but... should we restore the previous mode before autojoin the public chat? or is it intentionally set to false?

Reply:
It's intentionally set false to avoid make a commit for every new member. We set true again at `GroupChatRoom::syncMembers`